shop/src/main/java/ro/msg/learning/shop/model/Stock.java

```diff
@@ -0,0 +1,26 @@
+package ro.msg.learning.shop.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import ro.msg.learning.shop.model.ids.StockId;
+
+import javax.persistence.*;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+@IdClass(StockId.class)
+public class Stock {
+
+    @Id
+    @ManyToOne
+    private Product product;
+    @Id
```

Comment:
Its nice for readability to leave one line of space between your declarations. This can also be achieved with formatting settings and the just hitting CTRL + ALT + L